pkg/controller/operators/olm/operator.go

```diff
@@ -449,18 +450,31 @@ func newOperatorWithConfig(ctx context.Context, config *operatorConfig) (*Operat
 		}
 	}
 
-	labelObjects := func(gvr schema.GroupVersionResource, informer cache.SharedIndexInformer, sync queueinformer.LegacySyncHandler) error {
+	complete := map[schema.GroupVersionResource]bool{}
+	completeLock := &sync.RWMutex{}
```

Comment:
nit: RW lock that doesn't seem to use RLock(). Could probably just use regular Mutex.